storagenode/notifications/notifications.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package notifications
```

Comment:
Lets avoid a second package name with similar name in regards to PR #3659

Reply:
why do we care? it is the same as `satellite/orders` and `storagenode/orders` or `console` pkg